pkg/credentialprovider/gcp/metadata.go

```diff
@@ -298,3 +298,8 @@ func (g *containerRegistryProvider) Provide(image string) credentialprovider.Doc
 	}
 	return cfg
 }
+
+// truncateBlob is a sanitizing function
+func truncateBlob(blob []byte) []byte {
+	return blob[:16]
```

Comment:
I'd like to be able to list this as a sanitizing function for the analysis associated to [KEP-1933](https://github.com/kubernetes/enhancements/issues/1933).  Analysis configuration can be found at [hack/testdata/levee/levee-config.yaml](https://github.com/kubernetes/kubernetes/blob/master/hack/testdata/levee/levee-config.yaml).

I'd value any input you have in how that should be commented / documented here at the sanitizing function.